src/site/stages/build/process-cms-exports/transformers/helpers.js

```diff
@@ -19,7 +20,29 @@ function createMetaTag(type, key, value) {
   };
 }
 
+/**
+ * If `value` is a string, it will remove all `\r`, and `\t` characters from it.
+ *
+ * @return {string}
+ */
+function removeLinebreaks(value) {
```

Comment:
We talked about this yesterday. I still don't have a better name, but this one isn't the most accurate.